script.service.kodi.callbacks/resources/lib/publishers/log.py

```diff
@@ -39,7 +40,7 @@ def __init__(self, interval=100):
         self.interval = interval
 
     def run(self):
-        f = open(self.logfn, 'r')
+        f = codecs.open(self.logfn, 'r', encoding='utf-8', errors='ignore')
```

Comment:
OK, do I need to fix this before this pull can be merged? Or can it wait until the next revision?